schema.sql

```diff
@@ -9,3 +9,22 @@ CREATE TABLE animals (
 	weight_kg FLOAT,
 	species VARCHAR(255)
 );
+
+CREATE TABLE owners(
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	full_name VARCHAR(255),
+	age int
+);
+
+CREATE TABLE species(
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	name VARCHAR(255)
+);
```

Comment:
Please consider adding `NOT NULL` to your primary keys. 
Kindly note that adding a "NOT NULL" constraint to a primary key is a common practice in database design. A primary key is a unique identifier for each record in a database table, and by definition, it cannot contain NULL values. When you add the "NOT NULL" constraint to a primary key column, you are ensuring that every record in that table must have a valid, non-null value in that column.
Some people may argue that it is not necessary to add since some databases will not allow null values, but enforcing the "NOT NULL" constraint on a primary key column is a best practice in database design. It helps maintain data integrity, supports efficient querying, and simplifies database operations.